data/brands/office/telecommunication.json

```diff
@@ -396,6 +439,97 @@
         "name:zh": "中国联通",
         "office": "telecommunication"
       }
+    },
+    {
+      "displayName": "中國聯通 China Unicom",
+      "locationSet": {"include": ["hk"]},
+      "tags": {
+        "brand": "中國聯通 China Unicom",
+        "brand:en": "China Unicom",
+        "brand:wikidata": "Q1068485",
+        "brand:wikipedia": "zh:中国联通",
+        "brand:zh": "中國聯通",
+        "brand:zh-Hans": "中国联通",
+        "brand:zh-Hant": "中國聯通",
+        "name": "中國聯通 China Unicom",
+        "name:en": "China Unicom",
+        "name:zh": "中國聯通",
+        "name:zh-Hans": "中国联通",
+        "name:zh-Hant": "中國聯通",
+        "office": "telecommunication"
+      }
+    },
+    {
+      "displayName": "數碼通 SmarTone",
+      "locationSet": {"include": ["hk", "mo"]},
+      "tags": {
+        "brand": "數碼通 SmarTone",
+        "brand:en": "SmarTone",
+        "brand:wikidata": "Q11079498",
+        "brand:wikipedia": "zh:數碼通",
+        "brand:zh": "數碼通",
+        "brand:zh-Hans": "数码通",
+        "brand:zh-Hant": "數碼通",
+        "name": "數碼通 SmarTone",
+        "name:en": "SmarTone",
+        "name:zh": "數碼通",
+        "name:zh-Hans": "数码通",
+        "name:zh-Hant": "數碼通",
+        "office": "telecommunication"
+      }
+    },
+    {
+      "displayName": "新移動通訊 SUN Mobile",
+      "locationSet": {"include": ["hk"]},
+      "tags": {
+        "brand": "新移動通訊 SUN Mobile",
+        "brand:en": "SUN Mobile",
+        "brand:wikidata": "Q11080923",
+        "brand:wikipedia": "zh:新移動通訊",
+        "brand:zh": "新移動通訊",
+        "brand:zh-Hans": "新移动通讯",
+        "brand:zh-Hant": "新移動通訊",
+        "name": "新移動通訊 SUN Mobile",
+        "name:en": "SUN Mobile",
+        "name:zh": "新移動通訊",
+        "name:zh-Hans": "新移动通讯",
+        "name:zh-Hant": "新移動通訊",
+        "office": "telecommunication"
+      }
+    },
+    {
+      "displayName": "澳門電訊 CTM",
+      "locationSet": {"include": ["mo"]},
+      "tags": {
+        "brand": "澳門電訊 CTM",
+        "brand:en": "CTM",
+        "brand:pt": "CTM",
+        "brand:wikidata": "Q2990317",
+        "brand:wikipedia": "zh:澳門電訊",
+        "brand:zh": "澳門電訊",
+        "brand:zh-Hans": "澳门电讯",
+        "brand:zh-Hant": "澳門電訊",
+        "name": "澳門電訊 CTM",
+        "name:en": "CTM",
+        "name:pt": "CTM",
+        "name:zh": "澳門電訊",
+        "name:zh-Hans": "澳门电讯",
+        "name:zh-Hant": "澳門電訊",
+        "office": "telecommunication"
+      }
+    },
+    {
+      "displayName": "3",
```

Comment:
Maybe `3` don't fit the brand name on the street, according to common wikimedia's photo:

https://commons.wikimedia.org/wiki/File:3_HK_Shop_in_V_city_2015.jpg

<img src="https://upload.wikimedia.org/wikipedia/commons/3/38/3_HK_Shop_in_V_city_2015.jpg" width=400px>

Reply:
Thanks for your research. I changed the name and set it for HK & MO only. I think `Three.` is the shop name for HK & MO region.